README.md

```diff
@@ -1,122 +1,25 @@
-# PX4 Drone Autopilot
+## A PX4 Integrated Framework for Modeling and Controlling Multicopters with Tiltable Rotors
```

Comment:
Please don't replace the readme

Reply:
OK, sorry.
It was just the readme for the paper's code